sources/platform/integrations/ai/qdrant.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Qdrant integration
+description: Learn how to integrate Apify with Qdrant to feed data crawled from the web into the Qdrant vector database.
+sidebar_label: Qdrant
+sidebar_position: 4
+slug: /integrations/qdrant
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+**Learn how to integrate Apify with Qdrant to transfer crawled data into the Qdrant vector database.**
+
+---
+
+[Qdrant](https://qdrant.tech) is a high performance managed vector database that allows users to store and query dense vectors for next generation AI applications such as recommendation systems, semantic search, and retrieval augmented generation (RAG).
+
+The Apify integration for Qdrant enables you to export results from Apify Actors and Dataset items into a specific Qdrant collection.
+
+## Prerequisites
+
+Before you begin, ensure that you have the following:
+
+- A [Qdrant database](https://qdrant.tech) set up.
+- A Qdrant URL to the database and Qdrant API token.
+- An [OpenAI API key](https://openai.com/index/openai-api/) to compute text embeddings.
```

Comment:
```suggestion
- An [OpenAI API key](https://openai.com/index/openai-api/) to compute text embeddings. The actor can also be configured to use [Cohere](https://cohere.com/) embedding models.
```

Reply:
@Anush008 I'm sorry, I missed your comment when merging. I will include it in the next update. 

We are re-working the integration section in console.apify.com and we will need to re-work this doc anyway.